third_party/xla_client/BUILD

```diff
@@ -11,6 +10,10 @@ load(
     "@org_tensorflow//tensorflow/tsl/platform/default:cuda_build_defs.bzl",
     "if_cuda_is_configured",
 )
+load(
+    "//bazel:tensorflow.bzl",
+    "if_with_tpu_support",
```

Comment:
I wonder how or where the flag `if_true`/`if_false` is passed into this macro.

Reply:
At the call site, see below at [https://github.com/pytorch/xla/blob/0961d43f46a84c53e8a45bc9a583063a40188367/third_party/xla_client/BUILD#L138](usage).

Reply:
Thanks!